backend/main/templates/home.html

```diff
@@ -201,27 +203,30 @@ <h3>Debugging</h3>
                         <div class="feature-text" data-aos="fade-right" data-aos-duration="800" data-aos-delay="200">
                             <div class="feature-header">
                                 <div class="feature-line"></div>
-                                <h3>Simple Tracebacks</h3>
+                                <h3>Enhanced Error Messages</h3>
                             </div>
                             <p>
-                                Tracebacks are more helpful than usual, with several enhancements:
-                                <br>- Highlighting the exact operation that failed, not just the line, using executing
-                                <br>- Tables of local variables and simple expressions evaluated by pure_eval
-                                <br>- Suggestions for fixes provided by DidYouMean
-                                <br>- Beginner friendly explanations provided by friendly-traceback (shown when hovering over the little i icon)
-                                <br>- Showing multiline statements in full thanks to stack_data without showing unnecessary extra lines
+                                Normal Python tracebacks are intimidating to beginners.
+                                futurecoder removes the fear and makes them helpful instead.
+                                <ul>
+                                    <li>Cleaner design with clutter removed</li>
+                                    <li>Additional useful information</li>
+                                    <li>Novice-friendly explanations and suggestions for fixes</li>
+                                </ul>
```

Comment:
IMO, this can be left out completely. It's extra text that adds nothing that the image doesn't already demonstrate.